### PR TITLE
integrate google maps using react

### LINKED ISSUE #324: Integrate google maps using React in adding hospitals
For adding hospitals list, I will do integrate google maps in react so that it provide directions to the hospital address
and show a route to that and to provide live updates on distance and location.

### PR DESCRIPTION
**Pull request by subhangi2731**
#### Description

For adding hospitals list, I will do integrate google maps in react so that it provide directions to the hospital address
and show a route to that and to provide live updates on distance and location.

Issue FIXES #324

#### Describe the changes you've made

I have made Googlemap.js in react for giving the location in case of emergency and modified Gynac.js and package.json files and thus integrated google maps for location in react.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Checklist

- [x] I have added my name in the contributors list at the end of README.md file.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.
- [x] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)

### CODE REVIEW

**Comment by subhangi2731** [comment]
@khushishikhu please review and merge my pr

**Comment by subhangi2731** [comment]
@khushishikhu I will add it now

**Comment by subhangi2731** [comment]
Screenshot from 2021-05-22 10-50-00
Screenshot from 2021-05-22 10-50-21

**Comment by subhangi2731** [comment]
@khushishikhu please merge my pr

File: periodo/components/Gynae/GoogleMap.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+              <h1>{'Jeypore'}</h1>
+            </div>
+        </InfoWindow>
+      </Map>
+    );
+  }
+}
+ 
+export default GoogleApiWrapper({
+  apiKey: ('API_KEY')
```
Comment (khushishikhu) [inline periodo/components/Gynae/GoogleMap.js:23]:
```suggestion
  apiKey: ('MAP_API_KEY')
```

**Review by khushishikhu — CHANGES_REQUESTED** [review]

**Comment by khushishikhu** [comment]
@subhangi2731 Update your branch

**Comment by subhangi2731** [comment]
@khushishikhu done please merge the pr

**Pull request merged**